WORKSPACE

```diff
@@ -78,3 +78,16 @@ http_archive(
     strip_prefix = "freetype-2.9",
     urls = ["https://download.savannah.gnu.org/releases/freetype/freetype-2.9.tar.gz"],
 )
+
+http_archive(
+    name = "bazel_skylib",
+    sha256 = "97e70364e9249702246c0e9444bccdc4b847bed1eb03c5a3ece4f83dfe6abc44",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
+    ],
+)
+
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
```

Comment:
I'm really not familiar with bazel_skylib, but is the URL scheme really slash-slash-colon instead of colon-slash-slash?

Reply:
This is fortunately not a URL scheme, nor it's a URL in any way, shape or form.  Yes, this syntax is in fact what it is (I copied that part from the README).